doc/man/man3/notcurses_tabbed.3.md

```diff
@@ -0,0 +1,159 @@
+% notcurses_tabbed(3)
+% v2.2.3
```

Comment:
likewise, IIRC `notcurses(3)` has references to all the widgets, and you should add it there, along with a `SEE ALSO` at the bottom. whatever the other widgets do. if i'm making this up, just close this comment, no worries.

Reply:
notcurses(3) references most of the manpages but it lacks for example notcurses_tree(3) and notcurses_metric(3) for example. If it should mention all the man pages, then I can add others too.

Reply:
hrmmm i don't think so...from `notcurses.3.md`:

```
## Widgets

A few high-level widgets are included, all built atop ncplanes:

* **notcurses_fds(3)** for dumping file descriptors/subprocesses to a plane
* **notcurses_menu(3)** for menu bars at the top or bottom of the screen
* **notcurses_multiselector(3)** for selecting one or more items from a set
* **notcurses_plot(3)** for drawing histograms and lineplots
* **notcurses_progbar(3)** for drawing progress bars
* **notcurses_reader(3)** for free-form input data
* **notcurses_reel(3)** for hierarchal display of block-based data
* **notcurses_selector(3)** for selecting one item from a set
```

Reply:
oh it is missing tree though!

Reply:
added `tree` in master 7b61822a7

Reply:
metric is still missing :/

Reply:
metric isn't a widget, per se. it doesn't belong here =].

Reply:
oh i meant the `SEE ALSO` section

Reply:
huh. i only `SEE ALSO` pages i explicitly mention right now. maybe i should indeed link to all of them from there. i will ponder it. for now, i must get ready for work.